build.gradle

```diff
@@ -24,6 +24,7 @@ tasks.withType(JavaCompile) {
 }
 
 dependencies {
+    testCompile 'com.fasterxml.jackson.core:jackson-core:2.8.4'
```

Comment:
But that dependency is not explicit it just so happened that this library is in class path.
This now declares that we do directly require o.f.j.core functions in the code.